win32/src/win32clipboardmodule.cpp

```diff
@@ -557,17 +557,9 @@ static PyObject *py_get_clipboard_sequence_number(PyObject *self, PyObject *args
     DWORD rc;
     typedef HRESULT(WINAPI * PFNGetClipboardSequenceNumber)();
 
-    // @comm This method is not available on some early Windows (eg 95) machines.
     HMODULE hmod = LoadLibrary(TEXT("user32.dll"));
-    PFNGetClipboardSequenceNumber pfnGetClipboardSequenceNumber = NULL;
-    if (hmod)
-        pfnGetClipboardSequenceNumber =
-            (PFNGetClipboardSequenceNumber)GetProcAddress(hmod, "GetClipboardSequenceNumber");
-    if (pfnGetClipboardSequenceNumber == NULL) {
-        if (hmod)
-            FreeLibrary(hmod);
-        return PyErr_Format(PyExc_RuntimeError, "This version of Windows does not support this function");
-    }
+    PFNGetClipboardSequenceNumber pfnGetClipboardSequenceNumber =
+        (PFNGetClipboardSequenceNumber)GetProcAddress(hmod, "GetClipboardSequenceNumber");
```

Comment:
A null check still makes sense to me

Reply:
Rereading this code, I think I can just call `WinUser.h`'s `GetClipboardSequenceNumber` directly.
We already assume other methods from that header to be available, and `GetClipboardSequenceNumber` should be available since Win2k: https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-getclipboardsequencenumber

I rebuilt with the last commit without adding anything to my clipboard and it gave me the same result.